packages/ui/src/radio/RadioField.tsx

```diff
@@ -1,12 +1,31 @@
 import {
-  FormControl,
+  FormControl as FormControlMui,
   FormControlLabel,
   FormControlLabelProps,
-  FormHelperText,
+  FormHelperText as FormHelperTextMui,
   Radio,
   RadioProps,
 } from '@material-ui/core';
 import { forwardRef, ForwardRefExoticComponent, ReactNode } from 'react';
+import styled from 'styled-components';
+import { Color } from '../theme/Color';
+
+const FormControl = styled(FormControlMui)`
+  justify-content: center;
+`;
+
+const FormHelperText = styled(FormHelperTextMui)`
+  color: ${Color.Dark300};
+  font-size: 12px;
+  line-height: 16px;
+  font-weight: 400;
+  margin-left: 30px;
+  margin-top: 2px;
```

Comment:
@umrzoq-toshkentov Can you check if something can be broke by these changes in margins and paddings in CTMS/STMS, etc? Because there could be places where we set custom margins to achieve the same effect as you did.

Reply:
@timurhamitov no there won't be any style break because in design mentioned with helperText Radio component max height is 42px, now it stays 40px. With width there is not any changes just height and inner Radio component margin, fontSize and color updates. Anyway I will check it once more time.